src/libs/actions/BankAccounts.ts

```diff
@@ -409,14 +415,17 @@ function acceptACHContractForBankAccount(bankAccountID: number, params: ACHContr
 
 /**
  * Create the bank account with manually entered data.
- * @param plaidMask - scheme for Plaid account number
  */
-function connectBankAccountManually(bankAccountID: number, accountNumber?: string, routingNumber?: string, plaidMask?: string, policyID?: string) {
-    const parameters: ConnectBankAccountManuallyParams = {
+function connectBankAccountManually(bankAccountID: number, bankAccount: PlaidBankAccount, policyID: string) {
+    const parameters: ConnectBankAccountParams = {
         bankAccountID,
-        accountNumber,
-        routingNumber,
-        plaidMask,
+        routingNumber: bankAccount.routingNumber,
+        accountNumber: bankAccount.accountNumber,
```

Comment:
When is constant key used and when is hardcoded key used?

i.e. I see that `BANK_INFO_STEP_KEYS.ROUTING_NUMBER` was used in pages

Reply:
I don't think we need to put the constant here, as we're now in another object (ConnectBankAccountParams), where attributes could have been named differently